### ISSUE TITLE
feat/ add sks chart

### ISSUE THREAD

**Issue by simon-the-shark** [Solvro/mobile-topwr#378]
- this uses shuld reuse fetching data from the task: https://github.com/Solvro/mobile-topwr/issues/377
- but the parsing might be more specific for this case
- this should be a full-screen closable chart. a bit like the original site: https://live.pwr.edu.pl/sks/
- My very ruff visualization: ![Image](https://github.com/user-attachments/assets/21bade36-d76f-403b-9f91-790e34a819ce)
- but basically this can possily reuse the chart from parkings view? - if will look fine
- basically consult the team when you have first example UI in flutter

**Comment by simon-the-shark** [comment]
when you wait for the #377 to complete, you can start crafthing the ui on mocked data

**Comment by simon-the-shark** [comment]
possible that we'll get this + trend + parsing on the api side

**Comment by czaja307** [comment]
`https://sks-api.topwr.solvro.pl/api/v1/sks-users/today`
